locale/de/messages.json

```diff
@@ -765,10 +766,11 @@
   "pld.requiescat.suggestions.missed-confiteor.content": "",
   "pld.requiescat.table.note": "",
   "pld.requiescat.title": "Requiescat-Benutzung",
+  "pld.royal-authority.checklist.description": "",
+  "pld.royal-authority.checklist.name": "",
   "pld.royal-authority.title": "",
-  "pld.sword-oath.checklist.description": "",
-  "pld.sword-oath.checklist.name": "",
-  "pld.sword-oath.checklist.requirement.name": "",
+  "pld.sepulchre.checklist.requirement.name": "",
+  "pld.supplication.checklist.requirement.name": "",
   "pld.timeouts.confiteor.content": "",
```

Comment:
I think the locale need cleaned up a bit. Just replacing sword oath with royal authority should work imo.

Reply:
Locale files should not be edited directly.  The naming of the entries is controlled by the "id" field of the Trans tags used within module files.

Reply:
I believe the issue is that I had the same text 3 times but 3 different keys. For an example see the list of the screenshot of the Royal Authority Analysis in PR Description. I use the same key for all three now, which also cleans up the messages a bit.

Though I'm not entirely sure if it causes issues how I do it now. Is there a better way to "define" a Translation at one point and then "use" it at another?